flag_groups.go

```diff
@@ -180,12 +187,15 @@ func validateOneRequiredFlagGroups(data map[string]map[string]bool) error {
 
 		// Sort values, so they can be tested/scripted against consistently.
 		sort.Strings(set)
-		return fmt.Errorf("at least one of the flags in the group [%v] is required", flagList)
+		return &FlagGroupError{
+			flagList:      flagList,
+			flagGroupType: FlagsAreOneRequired,
+		}
 	}
 	return nil
 }
 
-func validateExclusiveFlagGroups(data map[string]map[string]bool) error {
+func validateExclusiveFlagGroups(data map[string]map[string]bool) *FlagGroupError {
```

Comment:
```suggestion
func validateExclusiveFlagGroups(data map[string]map[string]bool) FlagGroupError {
```

Please read this

https://github.com/spf13/cobra/pull/2266#discussion_r2074792615